{"uuid":"4479c70e-3b43-4538-93c5-8c3023ff4f48","name":"Scenario: User enters Invalid Python code and Presented with Error Message","children":["55fe0b90-e671-40ab-aab9-91f896d9ebe2"],"befores":[],"afters":[{"name":"hooks.Hook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1681608148421,"stop":1681608148421},{"name":"hooks.Hook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1681608148755,"stop":1681608148756},{"name":"hooks.Hook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1681608148880,"stop":1681608148881},{"name":"hooks.Hook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1681608148965,"stop":1681608148965},{"name":"hooks.Hook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1681608148988,"stop":1681608148988}],"start":1681608148283,"stop":1681608148989}